tutorandroid/templates/android/build/config/tutor.yaml

```diff
@@ -68,7 +68,10 @@ BRANCH:
   ALTERNATE_HOST: ''
 
 #Platform names
-PLATFORM_NAME: "{{ PLATFORM_NAME }}"
+# TODO: Temporarily set PLATFORM_NAME based on OPENEDX_COMMON_VERSION to 'OpenEdx' for compatibility with the Android app theme directory
+# (https://github.com/openedx/openedx-app-android expects 'OpenEdx'). This is addressed in PR https://github.com/openedx/openedx-app-android/pull/335,
+# which is merged into sumac.master and develop. Revert to "PLATFORM_NAME: '{{ PLATFORM_NAME }}'" once the pr changes are merged in master.
```

Comment:
The "master" branch of android app is actually "main". Please update the comment and then squash the commits.